include/common_structs.h

```diff
@@ -315,23 +315,27 @@ typedef struct ScriptInstance {
 
 typedef ScriptInstance* ScriptList[MAX_SCRIPTS];
 
+struct Entity;
```

Comment:
🤔

Reply:
Why is this needed? Doesn't `struct Entity` suffice or does cc hate us

Reply:
It prevents a really nasty warning that comes up all over the place, saying that the `struct Entity` in the function typedef is limited to only that scope or something